sensor_msgs/Image image
---
body_pose/BodyInfo[] body_poses